src/content/en/fundamentals/performance/_toc.yaml

```diff
@@ -83,6 +83,8 @@ toc:
           path: /web/fundamentals/performance/http2/
       - title: PRPL Pattern
         path: /web/fundamentals/performance/prpl-pattern/
+      - title: Resource Prioritization – Getting the Browser to Help You
+        path: /web/fundamentals/performance/resource-prioritization/
```

Comment:
Is there going to be additional stuff in this folder, or only this article? If only this article, please don't put it into it's own folder structure.

Reply:
Just this article. Moved it into its own thing, and added some prefixes to the images to keep things organised.